lxd/response/response.go

```diff
@@ -462,12 +553,17 @@ func (r *fileResponse) Render(w http.ResponseWriter) error {
 			rd = fd
 		}
 
+		// we apply a similar strategy as the single file case by wrapping
+		// each part's reader with a TimeoutReader. This allows us to manage TCP connection
+		// deadlines based on read activity for each part of the multipart response.
+		tr := NewTimeoutReader(rd, tcpConn, 1*time.Minute)
```

Comment:
Can't this be a `NewTimeoutReadSeeker()` too? Then you could get rid of `NewTimeoutReader()`.